int32 ping
int32 echo
float32 vref
int32 mux_ch
int32 pin
int32 n_encoder
float32 dist_per_tick
float32 speed
float32 ticks_per_rotation
float32 wheel_radius
float32 wheel_separation
float32 wheelbase
float32 width
int32 can_id
int32 motor_freq
string servo_type
string dio_type
int32 port
int32 port_a
int32 port_b
int32 front_left
int32 front_right
int32 rear_left
int32 rear_right
bool invert_front_left
bool invert_front_right
bool invert_rear_left
bool invert_rear_right
int32 left
int32 right
bool invert_left
bool invert_right
